string status
uint16 vcl_id
uint16 spot_id
